mitosheet/src/mito/components/icons/ConditionalFormatIcon.tsx

```diff
@@ -6,9 +6,9 @@ import React from 'react';
 const ConditionalFormatIcon = (): JSX.Element => {
     return (
         <svg width="38" height="30" viewBox="0 0 38 30" fill="none" xmlns="http://www.w3.org/2000/svg">
-            <rect x="4.5" y="6.57031" width="29" height="16" fill="#F4F4F4" stroke="black"/>
+            <rect x="4.5" y="6.57031" width="29" height="16" fill="#F4F4F4" stroke="var(--mito-text)"/>
             <rect x="4.5" y="11.5703" width="29" height="6" stroke="#797774"/>
-            <rect x="4.5" y="6.57031" width="29" height="16" stroke="black"/>
+            <rect x="4.5" y="6.57031" width="29" height="16" stroke="var(--mito-text)"/>
             <rect x="12" y="11.5703" width="6.5" height="6" fill="#9D6CFF" stroke="#6229AB"/>
             <rect x="12" y="6.57031" width="11.5" height="5" fill="#F0969A" stroke="#C33725"/>
             <rect x="12" y="17.5703" width="14" height="5" fill="#F0969A" stroke="#C33725"/>
```

Comment:
I know we wanna keep this red, but I could see making this a variable we can control. Even just making it `--mito-red` or something.